include/bootstrap.php

```diff
@@ -3,6 +3,14 @@
 // Used for performance calculations
 $dStartTime = microtime(true);
 
+// set locale
+$language_code = 'ru_RU.utf8';
```

Comment:
Instead of setting this to a default, maybe this could be done via browser locale detection and default to English. I may not understand 100% how you implemented this yet so maybe there is a better way than hardcoding this here.